src/test/java/lambdabook/FilterBooksService_MaxPrice_Test.java

```diff
@@ -0,0 +1,74 @@
+package lambdabook;
+
+import objectsBasicsMandatoryPracticalTasks.Books.Author;
+import objectsBasicsMandatoryPracticalTasks.Books.Book;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.hamcrest.CoreMatchers.*;
```

Comment:
a best practice - do not import with asterist, import single classes.
That's easily configured in IDEA: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports